lib/memo_wise.rb

```diff
@@ -178,7 +184,9 @@ def #{method_name}
             call_str = "(#{call_str})"
             fetch_key = method.parameters.map(&:last)
             fetch_key = if fetch_key.size > 1
-                          "[#{fetch_key.join(', ')}].freeze"
+                          fetch_key_init =
```

Comment:
I think setting the `fetch_key_init` within the `if/else` block that sets `fetch_key` is a little confusing. What would you think of:

```ruby
if fetch_key.size > 1
  fetch_key_init = "key = [#{fetch_key.join(', ')}].hash"
  fetch_key = "key"
else
  fetch_key = fetch_key.first.to_s
end
```

Reply:
Yep totally agree this is confusing. I was trying to just get something working before cleaning up the code as I still don't love where the code is even with your change (there's a lot of duplication around `"key"`, for example).

Do you have a sec to pair on this?